apps/web/utils/actions/ai-rule.ts

```
@@ -41,197 +40,195 @@ import { aiDiffRules } from "@/utils/ai/rule/diff-rules";
 
-  try {
-    result = await aiCreateRule(prompt, user, user.email);
-  } catch (error: any) {
-    return { error: `AI error creating rule. ${error.message}` };
-  }
+    let result: Awaited<ReturnType<typeof aiCreateRule>>;
 
-  if (!result) return { error: "AI error creating rule." };
+    try {
+      result = await aiCreateRule(prompt, user, user.email);
+    } catch (error: any) {
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)